api/queue/work/model_service_deployment.go

```diff
@@ -58,17 +56,8 @@ func (depl *ModelServiceDeployment) Deploy(job *queue.Job) error {
 		return err
 	}
 
-	endpointArg := jobArgs.Endpoint
-	endpoint, err := depl.Storage.Get(endpointArg.ID)
```

Comment:
We no longer read the database to get the endpoint data to be deployed. Instead, we are using the endpoint argument passed on by queue.Job. It should be safe to read from the job since the same replicas process the job (publisher and consumer for a job is in the same replicas) and we have locking when getting unprocessed job.